requirements.txt

```diff
@@ -44,6 +44,7 @@ PyMySQL==1.0.3
 pyOpenSSL==23.2.0
 pyparsing==3.0.9
 pypng==0.20220715.0
+PySocks==1.7.1
```

Comment:
i can't find where you use this new library

Reply:
i try to access telegram use socks and that needed this lib so i added